app/helpers/webhooks_helper.rb

```diff
@@ -17,4 +17,28 @@ def webhook_sources_for_select(sources, none: false)
   def sources_for_select(sources)
     sources.map { |source| [source.titleize, source] }.to_a
   end
+
+  def webhook_help_text(source)
+    case source
+    when 'ci_pipeline'
+      help_text = <<~HTML.html_safe
```

Comment:
maybe `CiPipelineController::HELP_TEXT` ?

Reply:
Since this string is used in only one place does it make sense to extract it to a constant? Then that's two files you have to update each time you want to add a new integration.